source/Expression/ExpressionSourceCode.cpp

```diff
@@ -252,6 +263,10 @@ bool ExpressionSourceCode::SaveExpressionTextToTempFile(
     }
     if (!success)
       FileSystem::Unlink(FileSpec(expr_source_path.c_str(), true));
+
+#if defined(_WIN32)
```

Comment:
this line and the two below should go away if we abstract temp filenames into llvm or the LLDB host layer.